agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -296,7 +339,7 @@ case ${key} in
         fi
         shift
         ;;
-    -ns|--namespace)
+    -ns|--namespace|--project)
```

Comment:
I don't see `--project` in the help message

Reply:
I'll add the option

Reply:
updated the help to include `--project`